dbt_meshify/main.py

```diff
@@ -105,22 +111,32 @@ def add_contract(select, exclude, project_path, selector, public_only=False):
     Adds a contract to all selected models.
     """
     path = Path(project_path).expanduser().resolve()
+    logger.info(f"Reading dbt project at {path}")
     project = DbtProject.from_directory(path)
     resources = list(
         project.select_resources(
             select=select, exclude=exclude, selector=selector, output_key="unique_id"
         )
     )
+
+    logger.info(f"Selected {len(resources)} resources: {resources}")
     models = filter(lambda x: x.startswith("model"), resources)
     if public_only:
         models = filter(lambda x: project.get_manifest_node(x).access == "public", models)
+    logger.info(f"Adding contracts to models in selected resources...")
     for model_unique_id in models:
         model_node = project.get_manifest_node(model_unique_id)
         model_catalog = project.get_catalog_entry(model_unique_id)
         meshify_constructor = DbtMeshModelConstructor(
             project_path=project_path, model_node=model_node, model_catalog=model_catalog
         )
-        meshify_constructor.add_model_contract()
+        logger.info(f"Adding contract to model: {model_unique_id}")
+        try:
+            meshify_constructor.add_model_contract()
+            logger.success(f"Successfully added contract to model: {model_unique_id}")
+        except Exception as e:
```

Comment:
I am OK with this for now to add debugging but I have 2 comments:
- what do we want the tool to do when there is an error? stop at the first error or skip the error and continue until the end (which is what we are doing with the `try/except`)
- if possible, we should be specific about the exceptions raised so we can add specific logic/logging: `Exception as e` is the generic one but we could try to be more specific. This is something that can be revisited later.